AsyncDisplayKit/ASCollectionView.h

```diff
@@ -91,6 +91,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) id<ASCollectionViewLayoutInspecting> layoutInspector;
 
 /**
+ * The ASInterfaceState of this collection view.
+ */
+@property (nonatomic, readonly) ASInterfaceState interfaceState;
```

Comment:
Is there any need to expose this is public API? I am asking because in 2.0, we are going to eliminate the ability to directly initialize this class, and the official interface will be the node subclasses, which already have this property defined. So it may make more sense for now to have just the range controller delegate method provide access to this information.